gcp-resources/README.md

```diff
@@ -49,6 +49,12 @@ spec:
 Additionally, the container name will only be discovered via the environment variable `CONTAINER_NAME`
 which much be included in the environment.
 
+## Usage with Manual-Instrumentation
+
+It is recommended to use this resource detector with the [OpenTelemetry Autoconfiguration SPI](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#resource-provider-spi). The GCP resource detector automatically provides the detected resources via the [autoconfigure-spi](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure-spi) SDK extension.
+
+For a reference example showcasing the detected resource attributes and usage with `autoconfigure-spi`, see the [Resource detection example](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/tree/main/examples/resource).
```

Comment:
what do you think of moving this example to https://github.com/open-telemetry/opentelemetry-java-examples?

Reply:
It seems like a good idea to me. I did not see any existing examples around resource-detection, what do you think about the following directory structure: 
```
/opentelemetry-java-examples/resource-detection/gcp
```

This would allow for addition of other resource detection specific examples.

Reply:
On second thought, might be more consistent to drop it at the root level.

```
/opentelemetry-java-examples/resource-detection-gcp
```

Reply:
> On second thought, might be more consistent to drop it at the root level

ya, it looks like all the existing modules are at the root level 👍

Reply:
@trask 
https://github.com/open-telemetry/opentelemetry-java-examples/pull/371

Reply:
Updated the link to the example.
Thank you for the suggestion @trask.